### PR TITLE
Add "sphinx-copybutton" extension

### PR DESCRIPTION
**Pull request by amotl**
Hi there,

having a sound implementation for copying code snippets from the documentation as outlined within #258 is a longstanding wish. This patch uses the excellent [sphinx-copybutton](https://github.com/executablebooks/sphinx-copybutton) extension as suggested by @msbt. Thank you!

With kind regards,
Andreas.

/cc @proddata, @hammerhead, @seut

### CODE REVIEW

**Comment by seut** [comment]
Very nice, lgtm, thanks!

File: src/crate/theme/rtd/crate/static/css/custom.css [hunk 1]
```diff
@@ -8,6 +8,11 @@
   margin-bottom: 64px;
 }
 
+a.copybtn {
```
Comment (msbt) [inline src/crate/theme/rtd/crate/static/css/custom.css:11]: can we add `top: .5em;` to that selector so the icon has the same space to top as it does to the right? ;)

Comment (amotl) [reply src/crate/theme/rtd/crate/static/css/custom.css:11]: Sure. I just amended this patch accordingly.

**Review by msbt — APPROVED** [review]
perfect, thanks!

**Pull request merged**